packages/@blockprotocol/graph/src/custom-element.ts

```diff
@@ -35,10 +49,36 @@ export abstract class BlockElementBase extends LitElement {
     graph: { type: Object },
   };
 
+  private updateDerivedProperties() {
+    const blockEntitySubgraph = this.graph?.blockEntitySubgraph;
+
+    if (blockEntitySubgraph) {
+      const rootEntity = getRoots<{
+        editionId: EntityEditionId;
+        element: RootEntity;
+      }>(blockEntitySubgraph)[0];
```

Comment:
It feels like you shouldn't be passing in this generic argument. Is it not encapsulated within `blockEntitySubgraph`?

Reply:
Yes, correct, thank you – removed this and another instance in https://github.com/blockprotocol/blockprotocol/pull/866/commits/8c9cb91791d702baafd54164ac2bea189b3b230d